### PR TITLE
New Ballot

### PR DESCRIPTION
**Pull request by NameThatFollows**

### CODE REVIEW

**Comment by joel99** [comment]
WOW

**Comment by evan10s** [comment]
@joel99 Don't go complaining about my 30 file PRs anymore

File: client/package.json [hunk 1]
```diff
@@ -2,43 +2,53 @@
...
-    "redux-thunk": "^2.3.0"
+    "@types/jest": "24.0.15",
+    "@types/node": "12.0.10",
+    "@types/papaparse": "^5.0.1",
+    "@types/react": "16.8.22",
+    "@types/react-bootstrap": "^0.32.17",
+    "@types/react-dom": "16.8.4",
+    "@types/react-redux": "^7.1.0",
+    "@types/react-router-dom": "^4.3.4",
+    "@types/socket.io-client": "^1.4.32",
```
Comment (evan10s) [inline client/package.json]: Types should be devDependencies

Comment (NameThatFollows) [reply client/package.json]: Resolved in latest commits

**Pull request merged**